docs/02.API-REFERENCE.md

```diff
@@ -30,6 +30,56 @@ Enum that contains JerryScript API value types:
 
 *New in version 2.0*.
 
+## jerry_object_type_t
+
+Enum that contains JerryScript **object** value types:
+
+ - JERRY_OBJECT_TYPE_NONE - Non object type
+ - JERRY_OBJECT_TYPE_SIMPLE - Simple JavaScript object without any internal property
+ - JERRY_OBJECT_TYPE_ARRAY - Array object
+ - JERRY_OBJECT_TYPE_PROXY - Proxy object
+ - JERRY_OBJECT_TYPE_FUNCTION - Function object (see [jerry_function_type_t](#jerry_function_type_t))
```

Comment:
Why not the getter functions are linked?

Reply:
Please describe more precisely what the questing is aiming for.

Reply:
You refer to the enum type (jerry_function_type_t), while I would refer to the getter function (get_function_type or something)

Reply:
I see, that's a good point.